src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.FrameworkReferenceResolution.targets

```diff
@@ -326,12 +326,14 @@ Copyright (c) .NET Foundation. All rights reserved.
     </PropertyGroup>
 
     <ResolveTargetingPackAssets FrameworkReferences="@(FrameworkReference)"
+                                ProjectLanguage="$(Language)"
                                 ResolvedTargetingPacks="@(ResolvedTargetingPack)"
                                 RuntimeFrameworks="@(RuntimeFramework)"
                                 GenerateErrorForMissingTargetingPacks="$(GenerateErrorForMissingTargetingPacks)"
                                 NuGetRestoreSupported="$(_NuGetRestoreSupported)"
                                 NetCoreTargetingPackRoot="$(NetCoreTargetingPackRoot)">
       <Output TaskParameter="ReferencesToAdd" ItemName="Reference" />
+      <Output TaskParameter="AnalyzersToAdd" Condition="'$(DisableFrameworkReferenceAnalyzers)' != 'true'" ItemName="Analyzer" />
```

Comment:
Should the `DisableFrameworkReferenceAnalyzers` property be documented somewhere?

Reply:
At the moment it's written here: https://github.com/dotnet/designs/blob/main/accepted/2021/InboxSourceGenerators.md

I think official docs should be added here:
https://github.com/dotnet/docs/blob/211a979fab01259e7c0ce8d1cd737b1ea4ba26f5/docs/core/project-sdk/msbuild-props.md

Reply:
https://github.com/dotnet/docs/pull/24896